src/motion_planning/src/plan_aak.cpp

```diff
@@ -0,0 +1,744 @@
+#include "ros/ros.h"
+#include <iomanip>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+#include <stdlib.h>
+#include <bits/stdc++.h>
+#include <std_msgs/String.h>
+#include <cmath>
+#include <nav_msgs/OccupancyGrid.h>
+#include <nav_msgs/MapMetaData.h>
+#include <visualization_msgs/Marker.h>
+#include <nav_msgs/Path.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/Quaternion.h>
+#include <tf/transform_broadcaster.h>
+#include <tf/tf.h>
+#include <limits.h>
+
+//x_max = 4.65= -6.9+COLUMNS*0.05;  //y_max = 3.8= -5.9+ROWS*0.05;
+
+# define ROWS 194
```

Comment:
prefer const variables over macros
Try to avoid macros as fas as possible. The is because macros are expanded blindly and may lead to unnecessary compile errors.
```suggestion
const std::size_t kRows = 194
```